CHANGES.md

```diff
@@ -8,6 +8,9 @@ Change Log
 ### 1.23 - 2016-07-01
 
 * Breaking changes
+    * Removed `castShadows` and `receiveShadows` properties from `Model`, `Primitive`, and `Globe`. Use `shadows` instead with the `ShadowMode` enum, e.g. `model.shadows = ShadowMode.ENABLED`.
```

Comment:
Throughout, these changes should be deprecated (over 1 release is fine), see https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/CodingGuide#deprecation-and-breaking-changes